### DISCUSSION TITLE
it's allways Failed upload，but it has been uploaded successfully.

### DISCUSSION THREAD

**Discussion by sunnyguhz** [stm32duino/Arduino_Core_STM32#2426] [Q&A] [answered]
**Describe the bug**
when I upload,it's allways to tell me that "Failed uploading: uploading error: exit status 1".But it has been uploaded successfully.

**To Reproduce**
any code

**Screenshots**
Memory Programming ...
Opening and parsing file: eeprom_clear.ino.bin
  File          : eeprom_clear.ino.bin
  Size          : 9004 Bytes
  Address       : 0x08000000 

Erasing memory corresponding to segment 0:
Erasing internal memory sectors [0 4]
Download in Progress:

File download complete
Time elapsed during download operation: 00:00:00.329

RUNNING Program ... 
  Address:      : 0x8000000
Application is running
Start operation achieved successfully
Failed uploading: uploading error: exit status 1

**Desktop (please complete the following information):**
 - OS: [Windows10 22H2]
 - Arduino IDE version: [2.3.2]
 - STM32 core version: [2.8.0]
 - Tools menu settings if not the default: [No Serial]
 - Upload method: [SWD]

**Board (please complete the following information):**
FQBN: STMicroelectronics:stm32:GenG0:pnum=GENERIC_G030C8TX,xserial=disabled

**Comment by fpistm** [comment]
Hi @sunnyguhz 
What is the version of STM32CubeProgrammer?
Is the reset signal connected?

**Reply by sunnyguhz** [reply]
The version of STM32CubeProgrammer is 2.6.0
It's all ways tell me Failed upload，whenever the reset signal is connected or not.
And it's no problem before 2.8.0.

**Reply by fpistm** [reply, marked as answer]
Update the STM32CubeProgrammer version and the STLink firmware.

**Reply by sunnyguhz** [reply]
OK，it's works all right now. Thank you.

**Reply by fpistm** [reply]
Fine. Thanks for the feedback since 2.8.0 the script managing the STM32CubeProgrammer has been hardened. So it could explain why you got this error as probably old version raised a minor error not catch by the script.
